app/src/main/java/com/example/drawingApp/Utils/ImageUtility.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.drawingApp.Utils
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.drawable.Drawable
+import android.net.Uri
+import androidx.activity.result.ActivityResultLauncher
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.target.CustomTarget
+import com.bumptech.glide.request.transition.Transition
+
+object ImageUtility {
+
+    /**
+     * launches with the purpose of obtaining an image, will open the users gallery
+     */
+    fun launchGalleryImageGetter(contentLauncher: ActivityResultLauncher<String>) {
+        contentLauncher.launch("image/*")
+    }
+
+    /**
+     * function that takes a Uri and gets a bitmap from it from the gallery.
+     */
+    fun getBitmapFromUriString(uri: Uri, context: Context, onBitmapLoaded: (Bitmap) -> Unit) {
```

Comment:
This isn't getting it from a `Uri String`, but from a `Uri`. Please rename

Reply:
renamed